packages/@yoda/manifest/index.js

```diff
@@ -6,9 +6,15 @@
  */
 
 var fs = require('fs')
+var path = require('path')
 var _ = require('@yoda/util')._
 
-var data = fs.readFileSync('/etc/manifest.json', 'utf8')
+var manifestPath = '/etc/manifest.json'
+if (process.env.YODA_RUN_MODE === 'host') {
+  manifestPath = path.join(__dirname, '../../..', manifestPath)
```

Comment:
It would be better to add a comment to describe what the real path looks like on host.